docker/sakai/placeholder.properties

```diff
@@ -275,6 +275,12 @@ calendar.export.previous.months = 6
 # Number of months in the future required in the export
 calendar.export.next.months = 12
 
+# Start the public announcement system
+pasystem.enabled=true
```

Comment:
Isn't the PA system enabled by default? It's already showing up for me on WL-11.